llvm/include/llvm/ADT/STLForwardCompat.h

```diff
@@ -36,6 +36,16 @@ template <typename T>
 using remove_cvref_t // NOLINT(readability-identifier-naming)
     = typename llvm::remove_cvref<T>::type;
 
+template <typename T>
```

Comment:
Can you add a TODO comment to use the std:: version when we bump the C++ version?